example-playbooks/async_extra/action_plugins/async_alias.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
```

Comment:
The previous version is actually the one recommended. 
Why did you have to change it?

Reply:
The doc page is referenced in the commit message